code/tasks/SiteTreeFullBuildEngine.php

```diff
@@ -137,7 +151,7 @@ protected function getAllLivePages() {
 			Translatable::disable_locale_filter();
 		}		
 		Versioned::reading_stage('Live');
-		$pages = DataObject::get("SiteTree");
+		$pages = DataObject::get("SiteTree")->sort('LastEdited');
```

Comment:
This kind of slipped into the commit!  But the rationale was that I wanted to process most recently changed pages first.